### ISSUE TITLE
grml-chroot via GNU/Screen: Unknown terminal type screen.xterm-256color

### ISSUE THREAD

**Issue by jkirk** [grml/grml-scripts#7]
When running `grml-chroot` via GNU/screen, the following happens:
```
root@grml ~ # grml-chroot /mnt
Writing /etc/debian_chroot ...
grml# reset                                                                                             
reset: unknown terminal type screen.xterm-256color
Terminal type? 
```
Backspace and TAB doesn't work either.
(Chrooting into a freshly installed Debian/stretch via `grml-debootstrap`.)

Running `grml-chroot` not in screen works fine.

I think this is more a screen-config problem but posting here anyway.

**Comment by mika** [comment]
Maybe related to https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=854414?

The broken backspace/tab keys are something we should investigate for sure, I also noticed that our GNU screen config lacks $USER inside the screen statusline.

**Comment by ft** [comment]
Screen cannot work properly if it can't find it's terminal description.
And (like tmux for that matter) *inside* of screen this has to be screen
or a variant of it. Though I'm unsure what screen.xterm-256color is. But
I'm not current with screen development so this maybe something recent.

Similarly, any applications that want to work with some aspects of a
terminal, like shells do, because they like keyboard input, they also
need to be able to find that description.

To me, this sounds like you need to install the package that includes
the terminfo database entry for screen.xterm-256color. And seems like
that is either ncurses-base or ncurses-term:

% apt-file search screen.xterm-256color
ncurses-base: /lib/terminfo/s/screen.xterm-256color
ncurses-term: /usr/share/terminfo/s/screen.xterm-256color

**Comment by akorn** [comment]
FWIW, this has been happening to me for a few years now when entering an older chroot from a newer host, inside `screen`, independent of `grml-chroot`; my solution is to install the `ncurses-term` package in the chroot.

**Comment by mika** [comment]
I can't reproduce this issue any longer, ncurses-base should be always present (verified with plain debootstrap as well as mmdebstrap) and it nowadays ships `/lib/terminfo/s/screen.xterm-256color`, so no need for `ncurses-term` any longer.

Quoting Debian's ncurses changelog:

```
ncurses (6.1+20180210-4) unstable; urgency=medium

[...]
  * Move screen.xterm-256color and rxvt-unicode-256color terminfo entries
    from ncurses-term to ncurses-base (Closes: #898666, #898948).
[...]
```

So Debian/buster and newer should be working just fine. Closing hereby (please re-open if you feel like we need and *can* do something about it anyway.)
